### ISSUE TITLE
[UX] Replace all instances of "Users" in the interface with "People"

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#5431]
I stumbled across the book settings page today and found the description text `Users with the "Administer book outlines" permission can add all content types.`.  This should say `People with the "Administer book outlines" permission can add all content types.`

There may be other instances of "Users" that needs to be replaced with "People". Perhaps we can get all (or at least many) of them at once.

**Comment by indigoxela** [comment]
I'm not sure if I agree here.

"Users" are people who use Backdrop (log in to do stuff). People are just people, including visitors. Changing Users to People throughout core, regardless of the context, is a regression IMO.

**Comment by olafgrabienski** [comment]
> Changing Users to People throughout core, regardless of the context, is a regression IMO.

Agreed! In many cases "people" works perfectly for "people who use Backdrop", like in `People with the "Administer book outlines" permission`. In other cases, the term "user" doesn't  stand for people but for user *accounts*.

**Comment by indigoxela** [comment]
Some odd examples for clarification:

- People account
- Delete people
- Create people
- People's signature
- People autocomplete
- People ID
- ...

:laughing:

**Comment by klonos** [comment]
I believe that @jenlampton mentioned something along these lines yesterday during the sprint session via zoom (paraphrasing, so blame my memory if inaccurate):

> We shouldn't use "users" when we mean "people", and if we are referring to accounts, then we should be using "user accounts".

...so your list above @indigoxela would actually become:
- user account
- delete user account
- create user account
- user account signature
- user account autocomplete
- user account ID (that's for the interface - code can still refer to these as `user ID`s or `UID`s)

"People" would be used in cases like the following:
- "when the user logs in" 👎🏼  -> should be "when people log in" instead
- "to allow people to log into the site, you need to create user accounts for them" 👍🏼 
- "anonymous site visitors and authenticated people"

In general, I think we should either use "people" or "user accounts" - avoid the words "user"/"users" on their own. For singular, we should use the word "person".

**Comment by klonos** [comment]
...but _I think_ that the combined term "end user" is OK(?).

**Comment by findlabnet** [comment]
>"anonymous site visitors and authenticated people" 👍🏼
"an anonymous site visitor and an authenticated person" 👍🏼

looks like KGB / CIA language - maybe it's just my personal historical deformation.

**Comment by indigoxela** [comment]
> I believe that @jenlampton mentioned something along these lines yesterday during the sprint session via zoom

@klonos Ah, I get it. This essential info isn't part of the issue description, though. Maybe clarify that to prevent further misunderstandings. Not everyone knows what somebody might have said in an internal Zoom meeting. :grinning: 

> looks like KGB / CIA language...

@findlabnet Ooops, really?

**Comment by RakeshPotnuru** [comment]
Hey, all the participants! I am overwhelmed seeing the number of issues this project has. So as part of the OpenForce 2022 program, I want to help as many issues as possible get closed.
So what about this issue? Users or People or have to change only some parts?

**Comment by jenlampton** [comment]
Yes, the words "users" and "people" are synonyms. There's no difference in
meaning.

People who are logged out are "anonymous users", so they are still users.

Beacuse people don't like being called users, and that's one of the things
that makes software feel robotic and unfriendly, we've already changed
nearly every instance of "users" to "people" in Backdrop core.

This issue is because we overlooked this one instance.

**Comment by stpaultim** [comment]
@RakeshPotnuru - If you look through the discussion so far, you will see the focus here is on replacing the word "User" with "People" when this makes sense. We will still refer to user accounts as user accounts. 

A good start on this issue would be to search through the code base and find a couple of examples, then post them here for feedback. I would not try to solve this problem all at once, until you have a better idea of what we mean. 

I did a quick search of core and found this:

```
Searching 69 files for "users" (case sensitive)

core/themes/bartik/templates/comment.tpl.php:
   32   *   - comment-by-node-author: Comment by the author of the parent node.
   33   *   - comment-preview: When previewing a new or edited comment.
   34:  *   The following applies only to viewers who are registered users:
   35   *   - comment-unpublished: An unpublished comment visible only to administrators.
   36   *   - comment-by-viewer: Comment by the user currently viewing the page.

core/themes/seven/template.php:
  159      $output .= '<nav class="breadcrumb">';
  160      // Provide a navigational heading to give context for breadcrumb links to
  161:     // screen-reader users. Make the heading invisible with .element-invisible.
  162      $output .= '<h2 class="element-invisible">' . t('You are here') . '</h2>';
  163      $output .= '<ol>';

core/themes/seven/css/responsive-tabs.css:
   17  .responsive-tabs-processed .duplicated-tab,
   18  .responsive-tabs-mobile ul.primary {
   19:   /* Hidden elements, hiding from sighted users only */
   20    visibility: hidden;
   21    height: 0;

core/themes/seven/css/style.css:
 1269  /* Add special icons for admin/config */
 1270  .admin-panel-people h3 {
 1271:    background-image: url('../images/users--black--64.png');background-image: url('../images/users--black--64.png');
 1272  }
 1273  .admin-panel-content h3 {
 ....
 1311  }
 1312  .block-dashboard-overview-user .block-title {
 1313:   background-image: url('../images/users--black--64.png');
 1314  }
 1315  .block-dashboard-updates .block-title {

core/themes/basis/template.php:
  127      $output .= '<nav class="breadcrumb">';
  128      // Provide a navigational heading to give context for breadcrumb links to
  129:     // screen-reader users. Make the heading invisible with .element-invisible.
  130      $output .= '<h2 class="element-invisible">' . t('You are here') . '</h2>';
  131      $output .= '<ol><li>' . implode('</li><li>', $breadcrumb) . '</li></ol>';

core/themes/basis/css/component/admin-tabs.css:
  184  .responsive-tabs-processed .duplicated-tab,
  185  .responsive-tabs-mobile ul.primary {
  186:   /* Hidden elements, hiding from sighted users only */
  187    visibility: hidden;
  188    height: 0;

core/themes/basis/templates/comment.tpl.php:
   32   *   - comment-by-node-author: Comment by the author of the parent node.
   33   *   - comment-preview: When previewing a new or edited comment.
   34:  *   The following applies only to viewers who are registered users:
   35   *   - comment-unpublished: An unpublished comment visible only to administrators.
   36   *   - comment-by-viewer: Comment by the user currently viewing the page.
``` 

WAS:
comment-by-viewer: Comment by the user currently viewing the page.
MY RECOMMENDATION:
comment-by-viewer: Comment by the person currently viewing the page.

WAS:
/* Hidden elements, hiding from sighted users only */
MY RECOMMENDATION:
/* Hidden elements, hidden from people without visual impairments */

WAS
// screen-reader users. Make the heading invisible with .element-invisible.
MY RECOMMENDATION:
// people using screen-readers. Make the heading invisible with .element-invisible.

WAS
background-image: url('../images/users--black--64.png');background-image: url('../images/users--black--64.png');
MY RECOMMENDATION:
<This is code - leave this as it is>

WAS
*   The following applies only to viewers who are registered users:
MY RECOMMENDATION:
*   The following applies only to site visitors with registered accounts:

**Comment by stpaultim** [comment]
NOTE: I see that @jenlampton originally specified the UI, so I don't know if we need to change the comments in code that I listed, but I don't think it hurts any.

**Comment by jenlampton** [comment]
Updating code comments is a good idea too, but since very few people will see those, my priority was for the UI. 

>  comment-by-viewer: Comment by the person currently viewing the page.

Love it

> / Hidden elements, hidden from people without visual impairments */

Love it

> // people using screen-readers. Make the heading invisible with .element-invisible.

Love it

> This is code - leave this as it is

Agree.

> The following applies only to site visitors with registered accounts:

Using 'viewers' here is important, since we're specifically talking about the people who are actively viewing the comment. How about: `The following applies only to viewers who have registered accounts:`

**Comment by manekasingh05** [comment]
Hi I would like to work on this issue under Codeflow openforce 2022 as I am new to contributing, Can you please provide me the required material and guide me along this.

**Comment by stpaultim** [comment]
@manekasingh05 - Welcome.

Here are two resources to get yourself started. 
https://backdropcms.org/news/openforce-2022-getting-started

If you are writing code and need to do local testing, you should see previous article for some information about setting up a local environment. For this specific issue, you could probably write the change in Github UI. 
https://backdropcms.org/news/creating-pull-requests-against-core-the-easy-noob-way

I already located the places these changes need to be made, see comment:
https://github.com/backdrop/backdrop-issues/issues/5431#issuecomment-1066175072

@jenlampton - please incorporate the suggested change in this comment
https://github.com/backdrop/backdrop-issues/issues/5431#issuecomment-1068625168

Please, let us know if you need more information or have additional questions.
